### PR TITLE
NF: Add alias argument to create_sibling_ria

### PR DESCRIPTION
**Pull request by mikapfl**
### Overview

Add an argument to create_sibling_ria to add an alias symlink to the RIA store to refer to this dataset.

### TODO

* [x] Integration test using create_sibling_ria directly
* [x] add to `.zenodo.json`

### Comment

Would be great to know if the way I'm adding this functionality looks sane, it is the first time I touch the datalad code.

### CODE REVIEW

File: datalad/customremotes/ria_utils.py [hunk 1]
```diff
@@ -204,3 +206,8 @@ def create_ds_in_store(io, base_path, dsid, obj_version, store_version):
     io.mkdir(archive_dir)
     io.mkdir(dsobj_dir)
     io.write_file(version_file, obj_version)
+    if alias:
+        alias_dir = base_path / "alias"
+        if not io.exists(alias_dir):
```
Comment (bpoldrack) [inline datalad/customremotes/ria_utils.py:211]: Conditional shouldn't be necessary. `io.mkdir` methods don't fail on existing dirs.

**Review by bpoldrack — COMMENTED** [review]
Thank you for approaching this, @mikapfl!

> Would be great to know if the way I'm adding this functionality looks sane, it is the first time I touch the datalad code.

Yes, this looks perfectly sane. :)

**Comment by mih** [comment]
This is very useful! Thx a lot!

File: datalad/customremotes/ria_utils.py (hunk 1 above)
Comment (mikapfl) [reply datalad/customremotes/ria_utils.py]: Cool, I dropped the conditional.

**Comment by mikapfl** [comment]
I've added a warning if the alias is already taken and tests for that. I'm not quite sure what's up with the `benchmarks` run or why codecov complains, I think these are unrelated to my changes.

**Comment by yarikoptic** [comment]
unrelated by promoted by this PR, @mikapfl please react to https://github.com/datalad/datalad-paper-joss/pull/93

**Comment by yarikoptic** [comment]
@bpoldrack @mih, please bless this PR -- I am not familiar with `ria` code to make a judgement call on either anything else is needed to get this merged

**Review by mih — APPROVED** [review]
Thx again, LGTM! I will merge this now. Over the next two days, I will give it a thorough real-world test ;-)

**Pull request merged**